### PR TITLE
feat(experiments): Make the time-duration consistent(in seconds) in all experiments

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

-  Make the time duration consistent in all experiments ( in seconds)

### CODE REVIEW

**Review by ksatchit — DISMISSED** [review]

File: charts/generic/pod-cpu-hog/engine.yaml [hunk 1]
```diff
@@ -29,7 +29,6 @@ spec:
               #verify the resources the app has been launched with
             - name: CPU_CORES
```
Comment (uditgaurav) [inline charts/generic/pod-cpu-hog/engine.yaml:33]: Can you add one comment `Container name where chaos has to be injected` above Target_container

File: charts/generic/pod-network-latency/engine.yaml [hunk 2]
```diff
@@ -24,19 +24,19 @@ spec:
...
-              value: 'eth0'                   
+              value: 'eth0'     
+
             - name: LIB_IMAGE
               value: 'gaiaadm/pumba:0.6.5'
+
             - name: NETWORK_LATENCY
-              value: '2000'
-            - name: TOTAL_CHAOS_DURATION
               value: '60000'
```
Comment (uditgaurav) [inline charts/generic/pod-network-latency/engine.yaml:39]: `NETWORK_LATENCY` Can't be converted?

File: charts/generic/pod-network-corruption/experiment.yaml [hunk 3]
```diff
@@ -51,7 +51,7 @@ spec:
       value: '100' #in PERCENTAGE
 
     - name: TOTAL_CHAOS_DURATION
-      value: '60000' #in ms
+      value: '60'
 
     # Period to wait before injection of chaos in sec
```
Comment (uditgaurav) [inline charts/generic/pod-network-corruption/experiment.yaml]: The time period to wait ... check for other RAMP TIME

**Review by uditgaurav — CHANGES_REQUESTED** [review]

File: charts/generic/pod-network-latency/engine.yaml (hunk 2 above)
Comment (ispeakc0de) [reply charts/generic/pod-network-latency/engine.yaml:39]: No it should be in ms

**Review by uditgaurav — APPROVED** [review]
LGTM

**Pull request merged**